### PR TITLE
AI-48: Resolve class disappearing issue by making styling more genera…

### PR DESCRIPTION
**Pull request by meladawy**
Styling for dashboard disappears when class "page-civicrm-dashboard" disappears. This resolve the issue by generalizing the styling and make it more abstract. 

## Before
screen shot 2017-05-31 at 12 37 14 pm

## After
screen shot 2017-05-31 at 12 38 20 pm

- [1] Tests Pass

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss [hunk 1]
```diff
@@ -20,3 +20,4 @@
 @import 'notifications';
 @import 'tooltip';
 @import 'wizard';
+@import 'widget-dashlet';
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss]: set alphabetic order

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 2]
```diff
@@ -0,0 +1,195 @@
+#crm-container {
+  .widget {
+    .widget-controls {
+      padding: 17px 10px !important;
+
+      h3 {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: Reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        font-family: $font-family-base;
+        font-size: $font-size-h2;
+      }
+
+      .fa-caret-down {
+        &:before {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: Reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+          content: "\f078";
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+        }
+      }
+
+      .fa-caret-right {
+        &:before {
+          content: "\f054";
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: same here

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+      table {
+        background: transparent;
+        border: none;
+        font-family: $font-family-base;
+
+        thead {
+          border-bottom: 1px solid $crm-dark-grayblue;
+
+          th {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+            vertical-align: middle;
+
+            div {
+              padding: 0;
+            }
+          }
+        }
+
+        tbody {
+          tr {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+
+  .portlet {
+    border: none;
+
+    .portlet-header {
+      background: $crm-accordion-header-color;
+      border-color: $crm-dark-grayblue;
+      -webkit-border-radius: 2px;
+      -moz-border-radius: 2px;
+      border-radius: 2px;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: please use border-radius variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+      color: $crm-dark-blue;
+      font-family: $font-family-base;
+      font-size: $font-size-medium;
+      font-weight: 600;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: please combined all font properties also use font-weight variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+    .crm-submit-buttons a.button {
+      background-color: $brand-primary;
+    }
+  }
+
+  div[id*="dashlets-header-col"] {
+    background: $crm-dasboard-header;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: can we use mixin for box-sizing ?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+      float: right;
+    }
+  }
+
+  div[id*="existing-dashlets-col"] {
+    background: $crm-white !important;
+    border: none !important;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: can we use mixin for box-sizing ?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+      &:last-child {
+        margin-bottom: 0;
+      }
+
+      .portlet-header {
+        background: $crm-accordion-header-color;
+        border-color: $crm-dark-grayblue;
+        -webkit-border-radius: 2px;
+        -moz-border-radius: 2px;
+        border-radius: 2px;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: please use border-radius variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+        color: $crm-dark-blue;
+        font-family: $font-family-base;
+        font-size: $font-size-medium;
+        font-weight: 600;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_widget-dashlet.scss]: please combined all font properties also use font-weight variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 14]
```diff
@@ -1,56 +1,157 @@
...
+  margin-top: 20px;
+
+  #civicrm-getting-started {
+    table {
+      tbody {
+        tr {
+          border-bottom: 0;
+
+          td {
+            &:nth-child(2) {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 15]
```diff
@@ -1,56 +1,157 @@
...
+  #civicrm-news-feed {
+    .ui-widget-header {
+      border-bottom-color: $crm-dark-grayblue;
+
+      .ui-tabs-active {
+        border-bottom: 2px solid $crm-white;
+        border-bottom-color: $crm-white;
+        border-color: $crm-dark-grayblue;
+
+        a {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 16]
```diff
@@ -1,56 +1,157 @@
...
+          font-weight: 400;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: Please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 17]
```diff
@@ -1,56 +1,157 @@
...
 
-    h1 {
-      &.page-title {
-        margin-top: -20px !important;
-        margin-bottom: 20px !important;
-        padding: 0 40px !important;
-        background: $crm-white;
+      a {
+        font-family: $font-family-base;
+        font-weight: 600;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 18]
```diff
@@ -1,56 +1,157 @@
...
 
-  .crm-submit-buttons {
-    margin: 30px 0;
+    [id*="civicrm-news-feed-"] {
+      padding: 0 !important;
+
+      .crm-accordion-wrapper {
+        &.collapsed {
+          .crm-accordion-header {
+            &:before {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 19]
```diff
@@ -1,56 +1,157 @@
...
+              color: $gray-darker;
+              content: "\f054";
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 20]
```diff
@@ -1,56 +1,157 @@
...
+              font-size: $font-size-base;
+            }
+          }
+        }
+
+        &:not(.collapsed) {
+          .crm-accordion-header {
+            &:before {
+              color: $gray-darker;
+              content: "\f078";
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 21]
```diff
@@ -1,56 +1,157 @@
...
+
+      .crm-accordion-header {
+        background: $crm-white !important;
+        border-bottom: 1px solid $crm-dark-grayblue;
+
+        .crm-news-feed-item-preview,
+        .crm-news-feed-item-title {
+          color: $crm-dasboard-header;
+          font-family: $font-family-base;
+          font-weight: 600 !important;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 22]
```diff
@@ -1,56 +1,157 @@
...
       }
     }
   }
 
-  .crm-dashboard-controls {
-    margin-top: 0;
+  .crm-activity-selector-dashlet {
+    .crm-accordion-header {
+      color: $gray-darker;
+      font-weight: 600 !important;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 23]
```diff
@@ -1,56 +1,157 @@
...
+      .dataTables_info {
+        padding: 10px 0;
+        width: 49%;
+      }
+
+      div[id*="s2id_autogen"] {
+        .select2-choice {
+          background: none;
+          border: 1px solid $crm-darker-grayblue;
+          border-radius: 2px;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use variable

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss (hunk 1 above)
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss]: Only properties should be sorted alphabetically. Sorting the @import may affect on the dependency

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 24]
```diff
@@ -1,56 +1,143 @@
...
+  #civicrm-news-feed {
+    .ui-widget-header {
+      border-bottom-color: $crm-dark-grayblue;
+
+      .ui-tabs-active {
+        border-bottom: 2px solid $crm-white;
+        border-color: $crm-dark-grayblue;
+        border-bottom-color: $crm-white;
+
+        a {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 25]
```diff
@@ -1,56 +1,143 @@
...
-      a {
-        padding: 0 !important;
+#civicrm-dashboard {
+  margin-top: 20px;
+
+  #civicrm-getting-started {
+    table tbody tr {
+      border-bottom: 0;
+
+      td:nth-child(2) {
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: reduce the level

File: org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss [hunk 26]
```diff
@@ -1,56 +1,143 @@
...
+    .ui-widget-header {
+      border-bottom-color: $crm-dark-grayblue;
+
+      .ui-tabs-active {
+        border-bottom: 2px solid $crm-white;
+        border-color: $crm-dark-grayblue;
+        border-bottom-color: $crm-white;
+
+        a {
+          font-weight: 400;
```
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/dashboard/_dashboard.scss]: please use font-weight variable

**Pull request merged**